crates/chain/src/keychain/txout_index.rs

```diff
@@ -285,7 +289,7 @@ impl<K: Clone + Ord + Debug> KeychainTxOutIndex<K> {
     /// This calls [`SpkTxOutIndex::net_value`] internally.
     ///
     /// [`sent_and_received`]: Self::sent_and_received
-    pub fn net_value(&self, tx: &Transaction, range: impl RangeBounds<K>) -> i64 {
+    pub fn net_value(&self, tx: &Transaction, range: impl RangeBounds<K>) -> SignedAmount {
```

Comment:
Why this needs to be a `SignedAmount`?

Reply:
It's because `net_value` used `i64` previously, and to keep the standard (and makes sense in my opinion given the  fn name) I used the `SignedAmount`, same for the other comments, they are all `net_value` references.

Reply:
Oh I see...